docs/kyma/docs/037-CRD.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Installer Custom Resource
+type: CRD
+---
+
+The installer Custom Resource (CR) file controls the Kyma installer, which is a proprietary solution based on the [Kubernetes operator](https://coreos.com/operators/). The file contains the basic information that defines Kyma installation. You can generate a custom CR file to install Kyma using specific parameters or a specific release in a `TAR.GZ` format.
+Find the custom resource template [here](../../../installation/resources/installer-cr.yaml.tpl).
+
+## Control the installation process
+
+To trigger the installation process, set the **action** label to `install` in the metadata of the Custom Resource with the installer configuration.
+To trigger the deinstallation process, set the **action** label to `uninstall` in the metadata of the Custom Resource with the installer configuration.
+
+## Generate a new Custom Resource file
+
+Use the `create-cr.sh` script to generate the Custom Resource file. The script accepts these arguments:
+
+- `--output` - mandatory. The location of the Custom Resource output file
+- `--url` - the URL of the Kyma package to install
+- `--version` - the Kyma version
+- `--ipaddr` - the load balancer IP
+- `--domain` - the instance domain
+
+For example:
+```
+$ ./installation/scripts/create-cr.sh --output kyma-cr.yaml --url {Kyma_TAR.GZ_URL} --version 0.0.1 --domain kyma.local
```

Comment:
you need to be more specific in URL part, the only official link we can provide is the one located here: https://github.com/kyma-project/kyma/releases/tag/latest

which is https://github.com/kyma-project/kyma/archive/latest.tar.gz
I don't see other option